letsencrypt/cli.py

```diff
@@ -1030,6 +1030,12 @@ def renew(config, unused_plugins):
     _renew_describe_results(config, renew_successes, renew_failures,
                             renew_skipped, parse_failures)
 
+    if renew_failures or parse_failures:
+        raise errors.Error("{0} renew failure(s), {1} parse failure(s)".format(
```

Comment:
Conclusion: it's caught by `_handle_exception` and will lead to an exit(1) without a traceback.

I think we should consider changing the convention that verbs return strings, and have exceptions be the only way that they return non-zero error codes.